### ISSUE TITLE
SSH stopped working on WSL2 Ubuntu 20.04 after latest Windows Updates

### ISSUE THREAD

**Issue by Karlheinzniebuhr** [microsoft/WSL#6199]
I used SSH on WSL2 just fine for months, however after the latest updates it stopped working. 
I'm running WSL2 on **Win10 version 2004 build 19041.610**
```
ssh -vvv asdf
OpenSSH_8.2p1 Ubuntu-4ubuntu0.1, OpenSSL 1.1.1f  31 Mar 2020
debug1: Reading configuration data /home/karl/.ssh/config
debug1: /home/karl/.ssh/config line 1: Applying options for *
debug1: /home/karl/.ssh/config line 7: Applying options for asdf
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: include /etc/ssh/ssh_config.d/*.conf matched no files
debug1: /etc/ssh/ssh_config line 21: Applying options for *
debug2: resolving "asdf.com.py" port 22
debug2: ssh_connect_direct
debug1: Connecting to asdf.com.py [192.168.1.1] port 22.
debug1: connect to address 192.168.1.1 port 22: Connection timed out
ssh: connect to host asdf.com.py port 22: Connection timed out
```
Windows Event log shows no errors nor hints.

**Comment by ad-on-is** [comment]
Do you have a 3rd-party firewall installed on Windows? I have problems reaching the internet in WSL2 with the Bitdefender firewall enabled

**Comment by Karlheinzniebuhr** [comment]
No third party antivirus, just windows defender and firewall. I tried turning it off but it makes no difference. Now it sometimes works and sometimes doesn't. I'll try to obtain some helpful log

**Comment by Biswa96** [comment]
You have to allow inbound connection to host Windows system in Windows Firewall.

**Comment by Karlheinzniebuhr** [comment]
I use ssh exclusively for outbound ssh connections, the firewall does not block outgoing connections, only incoming ones.

**Comment by Karlheinzniebuhr** [comment]
This is how I fixed the problem. My PC was on a different subnet than the Internet gateway. 
My PC's address: 192.168.0.3
Internet Gateway: 192.168.1.1
I changed my IP to 192.1.68.1.3 and the problem went away.
I had full Internet connectivity and no problem whatsoever with the old setup, but apparently it caused trouble for ssh on WSL2. The answer probably lies buried in how Win10 does networking with WSL2 or it is a problem with my home-router.
